Packs/HelloWorld/Integrations/FeedHelloWorld/FeedHelloWorld.py

```diff
@@ -0,0 +1,437 @@
+"""HelloWorld Feed Integration for Cortex XSOAR (aka Demisto)
+
+This feed integration is a good example on you can build a Cortex XSOAR feed
+using Python 3. Please follow the documentation links below and make sure that
+your feed integration follows the Code Conventions and required parameters, and passes the Linting phase.
+
+Developer Documentation: https://xsoar.pan.dev/docs/welcome
+Code Conventions: https://xsoar.pan.dev/docs/integrations/code-conventions
+Feed Required Parameters: https://xsoar.pan.dev/docs/integrations/feeds#required-parameters
+Linting: https://xsoar.pan.dev/docs/integrations/linting
+
+When building a Cortex XSOAR feed integration that is reusable, a lot of effort
+must be placed in the design. We recommend to fill a Design Document template,
+that allows you to capture Use Cases, Requirements and Inputs/Outputs.
+
+Example Design document for the this Integration (HelloWorld):
+https://docs.google.com/document/d/1wETtBEKg37PHNU8tYeB56M1LE314ux086z3HFeF_cX0
```

Comment:
This Design Doc is for the HelloWorld integration, not for the Feed Integration. Either remove these lines or produce a design document for the Feed. I think it's enough just to remove this as the Feed has less design considerations.

Reply:
As integrations and feeds has many similar guid lines I thought it'd be useful - but sure, removed as requested.